app/pages/admin/project-status.jsx

```diff
@@ -113,6 +123,20 @@ class ProjectStatus extends Component {
       .catch(error => this.setState({ error }));
   }
 
+  handleFeaturedProjectChange({ target }) {
+    const { featured, project } = this.state;
+    const featuredProject = featured[0];
+    if (featuredProject) {
+      featuredProject.update({ featured: false });
+      featuredProject.save()
+        .then(() => this.setState({ featured: featuredProject }))
```

Comment:
A couple of points here. `state.featuredProject` was an array before this update but is a project now. This will crash this function the next time it runs. 

Second question, should this update `state.featuredProject` to be the new featured project, not the old one?